docs/examples/State Estimation Example.ipynb

```diff
@@ -1085,7 +1085,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.12.2"
+   "version": "3.11.4"
```

Comment:
i highly recommend upgrading to python `3.12`. I'll make it optional, but strongly recommend